diff --git a/configure.ac b/configure.ac
index eb4d533..a5d16b2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -625,16 +625,16 @@ AS_CASE($enable_external_mediastreamer,
 	[PKG_CHECK_MODULES([MEDIASTREAMER], [mediastreamer])
 	MS2_VERSION=`$PKG_CONFIG --modversion mediastreamer`],
 	[no],
-	[AC_CONFIG_SUBDIRS( mediastreamer2 )
-	MEDIASTREAMER_DIR=${top_srcdir}/mediastreamer2
-	MEDIASTREAMER_CFLAGS="-I\$(top_srcdir)/mediastreamer2/include"
-	MEDIASTREAMER_LIBS="\$(top_builddir)/mediastreamer2/src/libmediastreamer_base.la \$(top_builddir)/mediastreamer2/src/libmediastreamer_voip.la"
+dnl	[AC_CONFIG_SUBDIRS( mediastreamer2 )
+dnl	MEDIASTREAMER_DIR=${top_srcdir}/mediastreamer2
+dnl	MEDIASTREAMER_CFLAGS="-I\$(top_srcdir)/mediastreamer2/include"
+dnl	MEDIASTREAMER_LIBS="\$(top_builddir)/mediastreamer2/src/libmediastreamer_base.la \$(top_builddir)/mediastreamer2/src/libmediastreamer_voip.la"
 dnl need to temporary change quotes to allow square brackets
-	changequote(<<, >>)
-	MS2_VERSION=`grep -e '^.C_INIT(' $MEDIASTREAMER_DIR/configure.ac | sed -e 's:\([^(]\+\)(\[mediastreamer\],\[\(.*\)\]):\2:g'`
-	changequote([, ])
-	MS2_DIR=mediastreamer2],
-	[AC_MSG_ERROR([bad value '${enable_external_mediastreamer}' for --enable-external-mediastreamer])]
+dnl	changequote(<<, >>)
+dnl	MS2_VERSION=`grep -e '^.C_INIT(' $MEDIASTREAMER_DIR/configure.ac | sed -e 's:\([^(]\+\)(\[mediastreamer\],\[\(.*\)\]):\2:g'`
+dnl	changequote([, ])
+dnl	MS2_DIR=mediastreamer2],
+dnl	[AC_MSG_ERROR([bad value '${enable_external_mediastreamer}' for --enable-external-mediastreamer])]
 )
 
 AC_SUBST(MEDIASTREAMER_CFLAGS)
@@ -717,24 +717,24 @@ AC_ARG_ENABLE(external-ortp,
 	[external_ortp=false]
 )
 
-if test "$external_ortp" = 'true'; then
+dnl if test "$external_ortp" = 'true'; then
 	PKG_CHECK_MODULES([ORTP], [ortp])
 	ORTP_VERSION=`$PKG_CONFIG --modversion ortp`
-else
-	AC_CONFIG_SUBDIRS( oRTP )
-	ORTP_CFLAGS="-I\$(top_srcdir)/oRTP/include"
-	ORTP_LIBS="\$(top_builddir)/oRTP/src/libortp.la"
-	if test x$ac_cv_c_bigendian = xyes ; then
-		ORTP_CFLAGS="$ORTP_CFLAGS -DORTP_BIGENDIAN"
-	fi
-	if test x$ntptimestamp = xtrue ; then
-		ORTP_CFLAGS="$ORTP_CFLAGS -DORTP_TIMESTAMP"
-	fi
-	ORTP_DIR=oRTP
-	changequote(<<, >>)
-	ORTP_VERSION=`grep -E ^[AC]+_INIT ${top_srcdir}/oRTP/configure.ac | sed -e 's:^.*_INIT(.*,\[\(.*\)\]):\1:g'`
-	changequote([, ])
-fi
+dnl else
+dnl	AC_CONFIG_SUBDIRS( oRTP )
+dnl	ORTP_CFLAGS="-I\$(top_srcdir)/oRTP/include"
+dnl	ORTP_LIBS="\$(top_builddir)/oRTP/src/libortp.la"
+dnl	if test x$ac_cv_c_bigendian = xyes ; then
+dnl		ORTP_CFLAGS="$ORTP_CFLAGS -DORTP_BIGENDIAN"
+dnl	fi
+dnl	if test x$ntptimestamp = xtrue ; then
+dnl		ORTP_CFLAGS="$ORTP_CFLAGS -DORTP_TIMESTAMP"
+dnl	fi
+dnl	ORTP_DIR=oRTP
+dnl	changequote(<<, >>)
+dnl	ORTP_VERSION=`grep -E ^[AC]+_INIT ${top_srcdir}/oRTP/configure.ac | sed -e 's:^.*_INIT(.*,\[\(.*\)\]):\1:g'`
+dnl	changequote([, ])
+dnl fi
 AC_SUBST(ORTP_CFLAGS)
 AC_SUBST(ORTP_LIBS)
 AC_SUBST([ORTP_VERSION])
